### PR TITLE
SCA: RHEL 9. Review section 1

### PR DESCRIPTION
**Pull request by mikykeane**
| Component | Action type           | Main Issue 
| --- | --- | ---
| SCA       | Rework |  #15461

# Description
This is the first Section of the SCA policy for RHEL 9 based on the CIS benchmark

## Main tasks

- [ ] Use the latest CIS benchmark PDF Draft version - CIS Red Hat Enterprise 9 Linux Benchmark v1.0.0
- [ ] Verify IDs numbers.
- [ ] Verify texts are correct: Title, Description, Rationale and Remediation.
- [ ] Verify Compliance: CIS, CIS_CSC.
- [ ] Verify condtion and rules:
  - [ ] To Pass.
  - [ ] To Fail.
- [ ] Solve Related issues:

### Syntax and semantic

- [ ] a) ID of each policy must be contiguous.
- [ ] b) The order and format set in [Documentation](https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/creating-custom-policies.html) must be respected.
- [ ] c) YML must be valid to avoid errors.

### Content

- [ ] a) Compare each check with its analog from CIS Benchmark.
- [ ] b) Try maintaining each rule as similar as possible with the `Audit` section from the CIS check.
- [ ] c) Check that the commands provide the expected output.
- [ ] d) When a failure is discovered, check similar policies to avoid repetition of the issue.

### Unit testing

- [ ] a) Output from `ossec.log` after the SCA scan and a raw output of the result of the checks.
<details><summary>Tests results</summary>
<p>

#### Analysisd (server or local)
analysisd.debug=2

#### Auth daemon debug (server)
authd.debug=0

#### Exec daemon debug (server, local, or Unix agent)
execd.debug=0

#### Monitor daemon debug (server, local, or Unix agent)
monitord.debug=0

#### Log collector (server, local or Unix agent)
logcollector.debug=0

#### Integrator daemon debug (server, local or Unix agent)
integrator.debug=0

#### Unix agentd
agent.debug=2

</p>
</details>

### Deployment

- [ ] a) If the policy it's new, it must be added to the `sca.files` templates.
- [ ] b) If the OS has many supported SCA policies, a policy must be set as the default policy. ([as example](https://github.com/wazuh/wazuh/blob/master/etc/templates/config/centos/sca.files))

### CODE REVIEW

File: ruleset/sca/rhel/9/cis_rhel9_linux.yml [hunk 1]
```diff
@@ -453,236 +527,259 @@ checks:
...
+      - nist_sp_800-53: ["AU-2"]
+      - cmmc_v2.0: ["AC.L2-3.1.7"]
+      - hipaa: ["164.312(b)","164.312(c)(1)","164.312(c)(2)"]
+      - pci_dss_3.2.1: ["10.2.1","11.5"]
+      - pci_dss_4.0: ["10.2.1","10.2.1.1"]
+      - soc_2: ["CC6.1"]
+      - mitre_techniques: ["T1565","T1565.001"]
+      - mitre_tactics: ["TA0001"]
     condition: all
     references: "AIDE stable manual: http://aide.sourceforge.net/stable/manual.html"
```
Comment (72nomada) [inline ruleset/sca/rhel/9/cis_rhel9_linux.yml:760]: The reference field should be an array:

Current:

```
  references: "AIDE stable manual: http://aide.sourceforge.net/stable/manual.html"
```


Expected:

```
  references: 
    - "AIDE stable manual: http://aide.sourceforge.net/stable/manual.html"
 ```

**Review by 72nomada — COMMENTED** [review]
Syntax and typos review Section 1

File: ruleset/sca/rhel/9/cis_rhel9_linux.yml (hunk 1 above)
Comment (mdiego92) [reply ruleset/sca/rhel/9/cis_rhel9_linux.yml]: Done.

**Comment by 72nomada** [comment]
Check: 1.1.2.1  Ensure /tmp is a separate partition (Automated)     

| Title comment: |  🟢    |     |     |
| Id:         |   🟢   |     |     |
| Title:         |   🟢   |     |     |
| Description:   |     |     |     |
| Rationale:     |     |     |     |
| Impact:        |    🔴  |   Missing   |     |
| Remediation:   |     |     |     |
| References:    |     |     |     |
| Compliance:    |     |     |     |
| Rules:         |     |     |     |

**Comment by 72nomada** [comment]
Section 1 - Up to check 1.1.5.4

  - <details><summary>1.1.5.1 Ensure separate partition exists for /var/log (Automated)
     🔴 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🔴  Add ':' after titles like 'Protecion from resource exhaustion'
            - Impact:         🔴  Missing
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🔴  Tactic is wrong, should be TA0040. PDF is wrong.
            - Rules:          🟢
    </details>

  - <details><summary>1.1.5.2 Ensure nodev option set on /var/log partition (Automated)
     🔴 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🟢
            - Impact:         ⚫
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🔴  Remove T1200.000 is not a valid technique.
            - Rules:          🟢
    </details>

  - <details><summary>1.1.5.3 Ensure noexec option set on /var/log partition (Automated)
     🟢 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🟢
            - Impact:         ⚫
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🟢
            - Rules:          🟢
    </details>

  - <details><summary>1.1.5.4 Ensure nosuid option set on /var/log partition (Automated)
     🟢 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🟢
            - Impact:         ⚫
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🟢
            - Rules:          🟢
    </details>

**Comment by 72nomada** [comment]
Section 1 - Up to check 1.1.6.4
  - <details><summary>1.1.6.1 Ensure separate partition exists for /var/log/audit (Automated)
     🔴 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🔴
              - There is a U+2019 char in attacker's. please replace it as needed            
            - Impact:         🔴  Missing
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🟢
            - Rules:          🟢
    </details>

  - <details><summary>1.1.6.2 Ensure noexec option set on /var/log/audit partition (Automated)
     🟢 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🟢
            - Impact:         ⚫
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🟢
            - Rules:          🟢
    </details>

  - <details><summary>1.1.6.3 Ensure nodev option set on /var/log/audit partition (Automated)
     🟢 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🟢
            - Impact:         ⚫
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🔴  Remove T1200.000 is not a valid technique.
            - Rules:          🟢
    </details>

  - <details><summary>1.1.6.4 Ensure nosuid option set on /var/log/audit partition (Automated)
     🟢 </summary>

            - Id:             🟢
            - Title:          🟢
            - Description:    🟢
            - Rationale:      🟢
            - Impact:         ⚫
            - Remediation:    🟢
            - References:     🟢
            - Compliance:     🟢
            - Rules:          🟢
    </details>

**Comment by 72nomada** [comment]
closing, completed by https://github.com/wazuh/wazuh/pull/16016

**Pull request closed without merging**